tests/memory_checker/test_memory_checker.py

```diff
@@ -224,9 +224,14 @@ def remove_stress_utility(duthost, container_name):
     logger.info("BGP sessions are started up.")
 
 
+@pytest.fixture(autouse=True)
+def rand_dut_for_all_functions(enum_rand_one_per_hwsku_frontend_hostname):
+    return enum_rand_one_per_hwsku_frontend_hostname
+
+
 @pytest.fixture
 def test_setup_and_cleanup(duthosts, creds, enum_dut_feature_container,
```

Comment:
iso modifying the test, can we update `generate_dut_feature_container_list` to generate the containers only the choosen dut.